.nycrc

```diff
@@ -1,6 +1,7 @@
 {
   "reporter": [
-    "lcov"
+    "lcov",
+    "html"
```

Comment:
This outputs a handy local coverage report in the `/coverage` folder that you can explore in your web browser to see all the coverage info.